pkg/metadata/host/host_tags.go

```diff
@@ -69,7 +69,7 @@ func getHostTags() *tags {
 
 	clusterName := clustername.GetClusterName()
 	if len(clusterName) != 0 {
-		hostTags = appendToHostTags(hostTags, []string{"cluster_name:" + clusterName})
+		hostTags = appendToHostTags(hostTags, []string{"cluster_name:" + clusterName, "kube_cluster_name:" + clusterName})
```

Comment:
In cases `cluster_name` conflicts with other integrations, should we add an option to remove this tag and keep only `kube_cluster_name`?
And in this case should put a warning somewhere advertising to activate this option and using this new tag, WDYT?

Reply:
yes could be useful to have the option to remove the tag, on it